### PR TITLE
More Transition lanes

### PR DESCRIPTION
**Pull request by acdlite**
Doesn't really have any effect now because we batch everything together anyway but will allow for more parallelism once we support that.

### CODE REVIEW

File: packages/react-reconciler/src/ReactFiberLane.new.js [hunk 1]
```diff
@@ -85,38 +85,54 @@ export const SyncLane: Lane = /*                        */ 0b0000000000000000000
...
+const InputDiscreteLane: Lanes = /*                     */ 0b0000000000000000000000000001000;
+
+const InputContinuousHydrationLane: Lane = /*           */ 0b0000000000000000000000000010000;
+const InputContinuousLane: Lanes = /*                   */ 0b0000000000000000000000000100000;
+
+export const DefaultHydrationLane: Lane = /*            */ 0b0000000000000000000000001000000;
+export const DefaultLane: Lanes = /*                    */ 0b0000000000000000000000010000000;
+
+const TransitionHydrationLane: Lane = /*                */ 0b0000000000000000000000100000000;
+const TransitionLanes: Lanes = /*                       */ 0b0000000000000111111111000000000;
```
Comment (rickhanlonii) [inline packages/react-reconciler/src/ReactFiberLane.new.js:97]: Shouldn't this have 14 bits on?

File: packages/react-reconciler/src/ReactFiberLane.new.js [hunk 2]
```diff
@@ -85,38 +85,54 @@ export const SyncLane: Lane = /*                        */ 0b0000000000000000000
...
+
+const RetryLanes: Lanes = /*                            */ 0b0001111000000000000000000000000;
+const RetryLane1: Lane = /*                             */ 0b0000001000000000000000000000000;
+const RetryLane2: Lane = /*                             */ 0b0000010000000000000000000000000;
+const RetryLane3: Lane = /*                             */ 0b0000100000000000000000000000000;
+const RetryLane4: Lane = /*                             */ 0b0001000000000000000000000000000;
+
+export const SomeRetryLane: Lane = RetryLane1;
+
+export const SelectiveHydrationLane: Lane = /*          */ 0b0001000000000000000000000000000;
```
Comment (rickhanlonii) [inline packages/react-reconciler/src/ReactFiberLane.new.js:121]: Is this supposed to be the same as RetryLane4?

File: packages/react-reconciler/src/__tests__/DebugTracing-test.internal.js [hunk 3]
```diff
@@ -16,6 +16,9 @@ describe('DebugTracing', () => {
 
   let logs;
 
+  const DEFAULT_LANE_STRING = '0b0000000000000000000000010000000';
+  const RETRY_LANE_STRING = '0b0000001000000000000000000000000';
```
Comment (rickhanlonii) [inline packages/react-reconciler/src/__tests__/DebugTracing-test.internal.js:20]: Nice

**Review by rickhanlonii — APPROVED** [review]

File: packages/react-reconciler/src/ReactFiberLane.new.js (hunk 1 above)
Comment (acdlite) [reply packages/react-reconciler/src/ReactFiberLane.new.js]: Dunno what you mean, I don't make mistakes

File: packages/react-reconciler/src/ReactFiberLane.new.js (hunk 2 above)
Comment (acdlite) [reply packages/react-reconciler/src/ReactFiberLane.new.js:121]: Check again. It was right all along.

**Pull request merged**